src/styles.css

```diff
@@ -10,6 +10,122 @@
 @import '@fortawesome/fontawesome-free/css/all.min.css';
 @import "~bootstrap-icons/font/bootstrap-icons.css";
 
+/* Direct CSS variable overrides - targeting Bulma's actual variables */
+:root {
+  /* Radius variables - Bulma's actual variable names */
+  --bulma-radius-small: 2px;
+  --bulma-radius: 2px;
+  --bulma-radius-medium: 6px;
+  --bulma-radius-large: 6px;
+  --bulma-panel-radius: 6px;
+
+  /* Color variables - Bulma's actual HSL system */
+  --bulma-primary-h: 229deg;
+  --bulma-primary-s: 53%;
+  --bulma-primary-l: 53%;
+  --bulma-info-h: 229deg;
+  --bulma-info-s: 53%;
+  --bulma-info-l: 53%;
+
+  /* Typography variables */
+  --bulma-size-normal: 0.9rem;
+  --bulma-body-font-size: 0.9rem;
+  --bulma-weight-medium: 400;
+  --bulma-weight-bold: 500;
+}
+
+/* Additional direct overrides for elements that don't fully use CSS variables */
+.button.is-danger {
+  background-color: #ff3860 !important;
+  border-color: #ff3860 !important;
+  color: white !important;
+}
+
+.button {
+  box-shadow: none !important;
+  font-weight: 400 !important;
+}
+
+.card,
+.card-header,
+.modal-card,
+.modal-card-head {
+  border-radius: 2px !important;
+}
+
+.panel {
+  border-radius: 3px !important;
+}
+
+.panel-heading {
+  border-radius: 3px 3px 0 0 !important;
+}
+
+.panel-block:last-child,
+.panel-tabs:last-child {
+  border-radius: 0 0 3px 3px !important;
+}
+
+.label:not(.lbl-child) {
+  font-weight: 500 !important;
+}
+
+.add-btn-small {
```

Comment:
What is this for? 
We had an add-btn class already? And why mixed in bulma overrides?